app/views/core/ModalComponent.coffee

```diff
@@ -0,0 +1,30 @@
+ModalView = require('./ModalView')
+store = require('core/store')
+silentStore = { commit: _.noop, dispatch: _.noop }
+
+module.exports = class ModalComponent extends ModalView
+  VueComponent: null # set this - will overwrite backbone modal
```

Comment:
Is this supposed to match `@vueComponent` below?

Reply:
We set the `VueComponent` which is instantiated and bound to `vueComponent`. Names could definitely be clearer.

Reply:
Used here: https://github.com/codecombat/codecombat/pull/4827/commits/99fbe6c8cec95f56267ff887eeffb5dbf2067688#diff-57f4f58f2fb0a2da66d076be6582d977R19